src/Aspire.Hosting.Azure/api/Aspire.Hosting.Azure.cs

```diff
@@ -191,6 +203,16 @@ public static T CreateExistingOrNewProvisionableResource<T>(AzureResourceInfrast
         public override string GetBicepTemplateString() { throw null; }
     }
 
+    public static partial class AzurePublisherExtensions
+    {
+        [System.Diagnostics.CodeAnalysis.Experimental("ASPIREAZURE001", UrlFormat = "https://aka.ms/dotnet/aspire/diagnostics#{0}")]
+        public static void AddAzurePublisher(this IDistributedApplicationBuilder builder, string name, System.Action<AzurePublisherOptions>? configureOptions = null) { }
+    }
+
+    public sealed partial class AzurePublisherOptions : Publishing.PublishingOptions
```

Comment:
This should also be experimental

Reply:
Done here - [Mark AzurePublisherOptions as Experimental (dotnet/aspire#8240)](https://github.com/dotnet/aspire/pull/8240/)